### PR TITLE
Stanley

### PR DESCRIPTION
**Pull request by stanley30543**

### CODE REVIEW

File: client/sidebar.jsx [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
       isWatched: false
     };
   }
 
   // when initializing the page
   componentDidMount() {
+    console.log(PATH);
     // send a get request for the product and seller infomation
-    axios.get('http://localhost:3210/item/0')
+    axios.get('http://localhost:3210/api/item/0')
```
Comment (kzlsakal) [inline client/sidebar.jsx:25]: We are still getting the first item in the database. Should we get the right item depending on what the URL path name is?

**Review by kzlsakal — CHANGES_REQUESTED** [review]
Hi Stanley, I added a comment where you are fetching the data from a hard-coded URL. I believe your API is set up to serve the dynamic requests. Please let me know if you have any questions.

File: client/sidebar.jsx (hunk 1 above)
Comment (stanley30543) [reply client/sidebar.jsx:25]: I can't put the dynamic path in it without having problem, so I'm still trying to figure out how to do it.

Comment (kzlsakal) [reply client/sidebar.jsx:25]: got it! I'll approve this and we can resolve that in a separate PR then.

**Review by kzlsakal — APPROVED** [review]

**Pull request merged**